pkg/ebpf/tracer.go

```diff
@@ -702,6 +714,26 @@ func (t *Tracer) determineConnectionDirection(conn *network.ConnectionStats) net
 	return network.OUTGOING
 }
 
+// initTCPCloseBatchMap initializes the tcp_close_batch map in eBPF By initializing it
+// in user-space we can save some precious bytes in the eBPF stack and increase the batch size.
+func (t *Tracer) initTCPCloseBatchMap() error {
+	// TODO: Replace this by the number of "real" number of CPU cores (as seen by the Kernel)
+	// (There is probably a utility in the process package with that)
+	const numCPUS = 64
```

Comment:
We've tried this in the past and ran into issues (look at PRs from @sfluor).

Thoughts on using 1024 since its the max map size that we define in the BPF code?

Reply:
Done